### DISCUSSION TITLE
Will Misskey support OAuth2?

### DISCUSSION THREAD

**Discussion by icecake0107** [misskey-dev/misskey#9329] [Q&A] [answered]
Hi there,

It looks like other instances use Mastodon/Pleroma/Friendica/PixelFed support OAuth2, is it something under develop or will it happen in Misskey?❓ 

Thanks!

**Comment by icecake0107** [comment, marked as answer]
I think I have learned what will happen, close.

**Comment by tamaina** [comment]
Yes, just currently syuilo is implementing.
